### ISSUE TITLE
[ERROR]: Uncaught panic! panicked at coeff.rs

### ISSUE THREAD

**Issue by iHichew** [asterinas/asterinas#870]
After booting Asterinas, the kernel will panic if you stay in the command line for about 20 minutes.
```
~ # [ERROR]: Uncaught panic!
panicked at /root/aster-ripos/kernel/libs/aster-util/src/coeff.rs:115:9:
assertion failed: rhs <= self.max_multiplier
printing stack trace:
   1: fn 0xffffffff886f4980 - pc 0xffffffff886f4998 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdbfe70;

   2: fn 0xffffffff886f4480 - pc 0xffffffff886f4937 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdbfe80;

   3: fn 0xffffffff88048030 - pc 0xffffffff8804803a / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc00d0;

   4: fn 0xffffffff88784800 - pc 0xffffffff8878484f / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc00e0;

   5: fn 0xffffffff88784930 - pc 0xffffffff88784975 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0130;

   6: fn 0xffffffff885ea590 - pc 0xffffffff885ea5ee / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc01a0;

   7: fn 0xffffffff884c8c80 - pc 0xffffffff884c8caa / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc01d0;

   8: fn 0xffffffff884c8cb0 - pc 0xffffffff884c8d7b / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0200;

   9: fn 0xffffffff884c9010 - pc 0xffffffff884c9033 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc02b0;

  10: fn 0xffffffff884cc570 - pc 0xffffffff884cc5a0 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc02d0;

  11: fn 0xffffffff884ca7b0 - pc 0xffffffff884ca7b9 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc02e0;

  12: fn 0xffffffff8837ce10 - pc 0xffffffff8837ce21 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0310;

  13: fn 0xffffffff8840d710 - pc 0xffffffff8840d728 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc03a0;

  14: fn 0xffffffff8861ee70 - pc 0xffffffff8861ef35 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc03c0;

  15: fn 0xffffffff886a7210 - pc 0xffffffff886a7228 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0440;

  16: fn 0xffffffff886b7060 - pc 0xffffffff886b7083 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0460;

  17: fn 0xffffffff886375b0 - pc 0xffffffff88637a09 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0480;

  18: fn 0xffffffff886a7250 - pc 0xffffffff886a7268 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0610;

  19: fn 0xffffffff886b7030 - pc 0xffffffff886b7053 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0630;

  20: fn 0xffffffff886e0ad0 - pc 0xffffffff886e0aeb / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0650;

  21: fn 0xffffffff88644760 - pc 0xffffffff886448fa / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0670;

  22: fn 0xffffffff886441a0 - pc 0xffffffff886441ce / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0730;

  23: fn                0x0 - pc 0xffffffff887491f4 / registers:

     rax                0x1; rdx 0xffffffff88810c00; rcx               0x16; rbx                0x0;
     rsi 0xffffffff88810b68; rdi 0xffff80007fdc0038; rbp                0x0; rsp 0xffff80007fdc0ab0;

make: *** [Makefile:138: run] Error 1
```

**Comment by cchanging** [comment]
Can you give the command to boot Asterinas? This panic usually occurs due to a long disable interrupt. Currently in main branch, using `make run` to boot Asterinas should not trigger this panic. If your usage is affected by this panic and you don't need to use the time module, you can simply remove this `assert`.

**Comment by iHichew** [comment]
I use `make run` command to launch Asterinas in the docker given by README.  This panic does not affect my usage. Thanks.

**Comment by junyang-zh** [comment]
Fixed by #1307
